support/sier-codec/src/lib.rs

```diff
@@ -48,13 +50,79 @@ impl Parser {
     pub fn struct_def(&self, name: &str) -> Option<&Arc<StructDef>> {
         self.structs.values().find(|s| s.type_name() == name)
     }
-}
 
-impl Default for Parser {
-    fn default() -> Self {
-        Parser {
-            structs: HashMap::new(),
+    fn transform_serde_value<'a>(
+        &mut self,
+        value: &SerdeValue,
+    ) -> Result<(Type, Value<'a>), Error<'a>> {
+        match value {
+            SerdeValue::Null => Ok((Type::Unit, Value::Unit)),
+            SerdeValue::Bool(_bool) => Ok((Type::Unit, Value::Unit)),
```

Comment:
Don't we have a bool type?

Reply:
We don't, but I guess I will implement it in this PR.